gewv_timeseries_client/timeseries_client.py

```diff
@@ -150,6 +150,24 @@ def get_dataframe(self, **kwargs):
 
         return df
 
+    def query_dataframe(
```

Comment:
You should reuse that function in the `get_dataframe` method. Save you the health check and the converting of the date time index.

Reply:
in get_dataframe werden versch. parameter übergeben.. in query_df nur ein string... ich könnte es in get_dataframe einbauen.. muss dann aber prüfen ob ein query zusätzlich übergeben wird... macht die fkt. aber auch wieder mehr als 2 zeilen länger ;)

Reply:
nope, the `build_query` function creates a string with the args. So you can reuse you function in the `get_dataframe` function easy.

Reply:
also, wenn flux_query = none -> build_query, sonst query=flux_query  ?

Reply:
okey I see, your on the wrong track. 
what I mean, you remove the health check and the date time conversion in the `get_dataframe` function and call there the `query_dataframe` like so.

```
    def get_dataframe(self, **kwargs):
        return self.query_dataframe(query=self.build_query(**kwargs))
```

Please make the cast now in the `query_dataframe` function, so that the type of the df is clearly a Dataframe.

Reply:
dann muss aber die build query umgebaut werden, da ich mit dem flux_query schon einen "fertigen" query übergeben und nicht erst vom client zusammenbauen lassen muss

Reply:
ist nun comitted... now-it's your turn